.github/workflows/build-and-publish.yml

```diff
@@ -14,19 +14,43 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: actions/setup-node@v1
-        with:
-          node-version: '10.17.0'
-          registry-url: https://npm.pkg.github.com/celonis
-
       - name: Yarn Install
         run: yarn install
 
       - name: Yarn Build
         run: yarn build
 
-      - name: Publish library
-        run: cd dist/ && npm publish --access public
+      - name: Move files to build
+        run: |
+          cp package.json dist/package.json
+          cp README.md dist/README.md
+          cp LICENSE dist/LICENSE
+
+      - name: Setup publish to Npm Registry
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.17.0'
+          registry-url: https://registry.npmjs.org/
+
+      - name: Publish to Npm Registry
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          CI: true
+        run: |
+          cd dist/
+          npm config set //registry.npmjs.org/:_authToken=$NODE_AUTH_TOKEN
```

Comment:
We can set this in the `Setup publish to Npm Registry` step instead.

I think something like this should do the trick (we might have to do it twice since you can't specify a registry in the publish command):
```
- uses: actions/setup-node@v1
  with:
    node-version: 10.x
    npm-registry:
      - registry: 'https://npm.pkg.github.com'
        scope: '@celonis'
        token: '${{secrets.GITHUB_TOKEN}}'
      - registry: 'https://registry.npmjs.org'
        token: '${{secrets.NODE_AUTH_TOKEN}}'
```

Reply:
I checked the `setup-node` implementation and the [registry-url](https://github.com/actions/setup-node/blob/v2.1.5/src/main.ts#L42) input is only a string so it wouldn't work.